playbooks/roles/l2tp-ipsec/handlers/main.yml

```diff
@@ -8,3 +8,7 @@
   service:
     name: xl2tpd
     state: restarted
+  register: xl2tpd_restart_result
```

Comment:
I'm not a huge fan of this particular work around. When using the f1-micro type on GCE this appears to work around https://github.com/jlund/streisand/issues/568.